api/rosetta/balance.go

```diff
@@ -48,36 +48,42 @@ func (d *Data) Balance(ctx echo.Context) error {
 	var req BalanceRequest
 	err := ctx.Bind(&req)
 	if err != nil {
-		return echo.NewHTTPError(http.StatusBadRequest, InvalidFormat(err.Error()))
+		return echo.NewHTTPError(http.StatusBadRequest, InvalidEncoding("request does not contain valid JSON-encoded body", err))
```

Comment:
Sounds like a candidate for error constant, no? I can see it 6 times just in this PR.

Reply:
We can move the string to a constant, but this is not actually a Go error, its a Rosetta error object. The Go error message goes into that object in the details map.